OSDOCS-12533: ABI on OCI GA (#84551)
openshift/openshift-docs: installing/installing_oci/installing-oci-agent-based-installer.adoc

In {product-title} {product-version}, you can use the Agent-based Installer to install a cluster on {oci-first}, so that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.

> Installing a cluster on {oci} is supported for both virtual machines (VMs) and bare metal machines.

Reviewer: ```suggestion
Installing a cluster on {oci} is supported for virtual machines (VMs) and bare-metal machines.
```

* link:https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/blockvolumeperformance.htm#vpus[Volume Performance Units (Oracle documentation)]
* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-about-instance-configurations.htm[Instance Sizing Recommendations for {product-title} on {oci} Nodes (Oracle documentation)]

[id="abi-oci-process-checklist_{context}"]
== Installation process workflow

> The following workflow describes a high level outline for the process of installing an {product-title} cluster on {oci} using the Agent-based Installer:

Reviewer: ```suggestion
The following workflow describes a high-level outline for the process of installing an {product-title} cluster on {oci} using the Agent-based Installer:
```

Reviewer: Should this intro explain that the list is showing where each step is performed with the items in parentheses?

It would be out of scope for this PR, but you might even think about doing this as a table since you have two columns of info here:

| Step      | Where       |
| ------------- | ------------- |
| 1. Create OCI resources and services | Oracle |
| 2. Disconnected environments: Prepare a web server that is accessible by OCI instances | Red Hat |
| 3. Prepare configuration files for the Agent-based Installer | Red Hat |

(etc)

Author: That's a solid idea actually, I will consider that. I don't want to set up a table while merging is the priority, but I think I am going to refine this doc later and can make this a table at that point.

* xref:../../installing/overview/installing-preparing.adoc#installing-preparing-selecting-cluster-type[Selecting a cluster installation type]
* xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#about-the-agent-based-installer[Preparing to install with the Agent-based Installer]
* xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-ocp-agent-retrieve_installing-with-agent-based-installer[Downloading the Agent-based Installer]

> * xref:../../disconnected/mirroring/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[Creating a mirror registry with mirror registry for Red{nbsp}Hat OpenShift]

Reviewer: ```suggestion
* xref:../../disconnected/mirroring/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[Creating a mirror registry with {product-mirror-registry}]
```
Optional - this would remove the "OpenShift" in plain text but that attribute seems to be capitalized incorrectly. Pick your poison :sweat_smile:
